tests/utils/verification/volumeproperties.go

```diff
@@ -109,17 +109,27 @@ func GetFullVolumeName(hostName string, volumeName string) string {
 // shorter name without @datastore suffix.
 func VerifyAttachedStatus(name, hostName, esxName string) bool {
 	log.Printf("Confirming attached status for volume [%s]\n", name)
+	return testAttachedStatus(name, hostName, esxName, esx.RetrieveVMNameFromIP(hostName))
+}
+
+// VerifyAttachedStatus - verify volume is attached and name of the VM attached
```

Comment:
Thats what I wanted to do but to start with its changes in multiple places and wanted to get a review before changing anything more.

The UUID needs to be passed or a new struct created to pass VM IP, VM name, ESX name and VM UUID.